src/main/java/lutchat/Parser.java

```diff
@@ -0,0 +1,96 @@
+package lutchat;
+
+public class Parser {
+    public static boolean parse(String userInput, Ui ui, TaskList taskList, Storage storage) {
+        String[] userInputArr = userInput.split(" ");
+        String keyword = userInputArr[0];
+
+        switch (keyword) {
+            case "bye":
+                return false;
+
```

Comment:
Switch case does not follow the coding standard. There should not be an indent for the switch case.